modules/tinymce/src/core/main/ts/DragDropOverrides.ts

```diff
@@ -250,8 +258,8 @@ const bindFakeDragEvents = function (editor: Editor) {
   });
 };
 
-const blockIeDrop = function (editor: Editor) {
-  editor.on('drop', function (e) {
+const blockIeDrop = (editor: Editor) => {
+  editor.on('drop', (e: MouseEvent) => {
```

Comment:
The types here are incorrect. This should be `EditorEvent<DragEvent>` as it's an editor event which means it has additional type information.

Actually, on a second look, this shouldn't actually even be here as it'll infer them.

- [ ] Remove incorrect types and let it infer the correct types